src/webextension/tests/testServer.h

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <QObject>
+
+class TestServer final : public QObject {
+  Q_OBJECT
+
+ private slots:
+  void testCanConnect();
+};

```

Comment:
probably should still include the ifdef preprpcessor directive just in case.
```
#ifndef TESTSERVER_H
#define TESTSERVER_H

// Declarations of functions, classes, and other entities

#endif // TESTSERVER_H
```